### ISSUE TITLE
get_cid service not available

### ISSUE THREAD

**Issue by bru-dot-edu** [ropensci/webchem#337]
---

I am having trouble with get_cid()

Input:
> get_cid("Triclosan")
Output:
 Service not available. Returning NA.
Error in get_cid("Triclosan") :

**Comment by Aariq** [comment]
Thanks.  We are aware of this issue and are currently working on a fix for it.

Duplicate of #336

**Comment by bru-dot-edu** [comment]
Oh that's great. Sorry I didn't notice the other submission. 

All of this is way over my head, but is this fix a day, week, or month long fix? Curious if I should wait it out or find an alternative solution. 

Many thanks for all of your help.

**Comment by Aariq** [comment]
I think it's actually fixed in a development branch, I'm just hoping for one of the other `webchem` devs to take a look at my changes before merging them.  If you'd like to try it out **now**, you can install directly from that branch with 
```r
devtools::install_github("ropensci/webchem", ref = "ping")
```
Just be warned that my changes haven't been reviewed yet, and remember that once the pull request is merged, you should probably re-install the development version from the main branch with `devtools::install_github("ropensci/webchem")`

**Comment by bru-dot-edu** [comment]
I tried installing on your unmerged branch and got this error: 

`Error in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]) : 
  namespace ‘cli’ 2.3.1 is already loaded, but >= 2.4.0 is required`

**Comment by Aariq** [comment]
This should be fixed now in the development version of `webchem`.  You can install it with `remotes::install_github("ropensci/webchem")`

**Comment by bru-dot-edu** [comment]
This is still not working for me.
